src/redux/custom-label.js

```diff
@@ -0,0 +1,29 @@
+import model from './model';
+
+class CustomLabel {
+  id = '';
+  displayLabel = '';
+  displayOnFullTextFinder = false;
+  displayOnPublicationFinder = false;
+
+  serialize() {
+    const data = {
+      type: this.type,
+      attributes: {},
+    };
+
+    data.attributes = {
+      id: this.id,
```

Comment:
I don't think `id` should be inside `data.attributes`. It should be placed next to `type` and `attributes` fields of `data`

Reply:
I'm not sure about it, according to API documentation `id` present just in attributes and does not exist in `data`

doc: https://s3.amazonaws.com/foliodocs/api/mod-kb-ebsco-java/p/custom-labels.html#eholdings_custom_labels_get